### PR TITLE
Added MMDetection guide

### PR DESCRIPTION
**Pull request by kapil-varshney**
Adds MMDetection guide

### CODE REVIEW

**Comment by priyankatuteja** [comment]
@guneetmutreja  and @kapil-varshney Changes suggested to Guneet offline. Once incorporated in MMSegmantation guide, you can add the missing ones to this guide too.

**Comment by kapil-varshney** [comment]
@guneetmutreja @priyankatuteja Can you please review?

File: guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb [hunk 1]
```diff
@@ -0,0 +1,226 @@
+{
```
Comment (guneetmutreja) [inline guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb:1]: <p>MMdetection is an open-source library with many popular and <strong>state-of-the-art</strong> object detection models. Through <code>arcgis.learn</code> we have provided a bridge to use the growing list of models provided by <strong>the</strong> MMDetection library. </p><p> </p><p>The MMdet team won the COCO 2018 object detection challenge. Their codebase, which is built with Pytorch, gradually evolved to include various models and methods. Although the detection task can be complex, the creators of MMDetection decomposed the models into different general components - Backbone, Neck, DenseHead, ROIExtractor<strong>,</strong> and ROIHead. With this abstraction the library is able to provide a multitude of single-stage (e.g. GHM, FCOS), two-stage (e.g. Double-Head R-CNN), multi-stage (e.g. Cascade R-CNN)<strong>,</strong> and other kinds of detection models. This abstraction-based approach also allows the development of models by changing the different components or even adding new ones. Read the MMDetection <a href="https://arxiv.org/pdf/1906.07155.pdf" target="_blank">paper</a> or checkout their <a href="https://github.com/open-mmlab/mmdetection" target="_blank">github repository</a> for more detail.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (guneetmutreja) [inline guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb:1]: <p>Let's change the world imagery from input imagery parameter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

**Review by guneetmutreja — CHANGES_REQUESTED** [review]
Please incorporate the changes.

**Comment by kapil-varshney** [comment]
> Please incorporate the changes.

Done. Thanks for the suggestions.

**Review by guneetmutreja — APPROVED** [review]
LGTM

**Comment by guneetmutreja** [comment]
@BP-Ent Can you help to review the guide?

File: guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb (hunk 1 above)
Comment (BP-Ent) [inline guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb:1]: <p>MMdetection is an open-source library containing many popular and state-of-the-art object detection models. Through <code>arcgis.learn</code>, we have provided a bridge to use the growing list of models provided by the MMDetection library. </p><p>In 2018, the MMdet team won the COCO object detection challenge. Their codebase, which is built with Pytorch, has gradually evolved to include various models and methods. Although the detection task can be complex, the creators of MMDetection decomposed the models into different general components - Backbone, Neck, DenseHead, ROIExtractor, and ROIHead. With this abstraction, the library is able to provide a multitude of single-stage (e.g. GHM, FCOS), two-stage (e.g. Double-Head R-CNN), multi-stage (e.g. Cascade R-CNN), and other detection models. This abstraction-based approach also allows the development of models either by changing the different components or even by adding new ones. Read the MMDetection <a href="https://arxiv.org/pdf/1906.07155.pdf" target="_blank">paper</a>, or checkout its <a href="https://github.com/open-mmlab/mmdetection" target="_blank">github repository</a>, for more details.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (BP-Ent) [inline guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb:1]: <p>Follow the steps <a href="https://developers.arcgis.com/python/guide/install-and-set-up/#Install-deep-learning-dependencies" target="_blank">here</a> to install deep learning dependencies in ArcGIS Pro or your Anaconda environment. </p><p>ArcGIS Pro 2.8 users additionally need to install the CUDA toolkit (version 11), mmcv-full, and mmdet libraries. Follow these steps to do so: </p><ul><li>Download and install the latest CUDA toolkit version from&nbsp;<a href="https://developer.nvidia.com/cuda-downloads" target="_blank">here</a>.</li><li>Add the installed CUDA toolkit's bin folder path (typically, C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.0\bin) to the (user or system) Path Environment Variables.</li><li>Run the following command in a cloned environment:</li><li><code>conda install -c esri mmcv-full mmdet</code></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (BP-Ent) [inline guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb:1]: <p>Using MMDetection with arcgis.learn is as simple as using any other object detection model in the library. The only additional step is providing the name of the model to be used when initializing the MMDetection model object.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (BP-Ent) [inline guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb:1]: <p>The parameters required are: </p><ul><li><code>data</code>&nbsp;- the data object prepared using&nbsp;<code>prepare_data</code></li><li><code>model</code>&nbsp;- the name of the models from the list of supported models.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (BP-Ent) [inline guide/14-deep-learning/Using MMDetection with `arcgis.learn`.ipynb:1]: <p>For more information about the API, visit the <a href="https://developers.arcgis.com/python/api-reference/arcgis.learn.toc.html#mmdetection" target="_blank">API reference for MMDetection</a>. For a detailed object detection workflow, refer to a sample <a href="https://developers.arcgis.com/python/sample-notebooks/detecting-and-categorizing-brick-kilns-from-satellite-imagery/" target="_blank">notebook</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

**Review by BP-Ent — CHANGES_REQUESTED** [review]
Suggested changes made on reviewnb

**Comment by mohi9282** [comment]
@kapil-varshney - any updates on the requested changes? Changes need to be reviewed and merged ASAP as the guide is scheduled for V1.9.1.

cc: @AtmaMani @priyankatuteja

**Comment by mohi9282** [comment]
removing from v1.9.1 milestone per discussion with Kapil.

**Comment by kapil-varshney** [comment]
@priyankatuteja Renamed the guide.

**Review by priyankatuteja — APPROVED** [review]

**Review by priyankatuteja — APPROVED** [review]

**Pull request merged**